posthog/api/test/__snapshots__/test_action.ambr

```diff
@@ -338,13 +338,7 @@
   INNER JOIN "posthog_team" ON ("posthog_action"."team_id" = "posthog_team"."id")
   LEFT OUTER JOIN "posthog_user" ON ("posthog_action"."created_by_id" = "posthog_user"."id")
   WHERE (NOT "posthog_action"."deleted"
-         AND ("posthog_action"."team_id" =
-                (SELECT U0."parent_team_id"
-                 FROM "posthog_team" U0
-                 WHERE U0."id" = 99999
-                 LIMIT 1)
-              OR ("posthog_team"."parent_team_id" IS NULL
-                  AND "posthog_action"."team_id" = 99999))
+         AND "posthog_action"."team_id" = 99999
```

Comment:
This shows me the `TeamRootMixin` is not in effect anymore, which is bad!

Reply:
I completely borked the teammixin import. wasn't inheriting right. This should all be sorted now